airflow/cli/commands/provider_command.py

```diff
@@ -64,3 +63,25 @@ def provider_get(args):
 def providers_list(args):
     """Lists all providers at the command line"""
     print(_tabulate_providers(ProvidersManager().providers.values(), args.output))
+
+
+def _tabulate_hooks(hook_items: Tuple[str, Tuple[str, str]], tablefmt: str):
```

Comment:
There is something wrong with the type of this parameter. I guess this is going to be a list of items because then we iterate over it. 
> hook_item[1][3]

Then we read the index of the element which probably does not exist. Am I right?

Reply:
Yeah. I just rebased it after some changes in the parent PR and did not yet fix all errors :)